guide/glossary.md

```diff
@@ -158,7 +158,7 @@ BIP44 instroduced the following structure, which BIP49 and BIP84 follow:<br/>
 The path to the first address in a bitcoin-wallet using BIP84 will look like this:<br/>
 `m/84h/0h/0h/0/0`
 
-For full interoperability a wallet should support all of these standards. More information can be found [here](https://learnmeabitcoin.com/technical/derivation-paths).
+For full [interoperability](/guide/foundations/principles/#interoperability) a wallet should support all of these standards. More information can be found [here](https://learnmeabitcoin.com/technical/derivation-paths).
```

Comment:
We could also link to the [Wallet interoperability page](http://localhost:4000/guide/foundations/wallet-interoperability/) in Foundations.

Reply:
@GBKS  Would you prefer that I link to that page instead?

Reply:
Could also add it in the more information sentence.

`More information can be found [here](https://bitcoin.design/guide/foundations/wallet-interoperability/#wallet-import-and-export) and  [here](https://learnmeabitcoin.com/technical/derivation-paths).`

Reply:
Done https://github.com/BitcoinDesign/Guide/pull/218#pullrequestreview-634496189